src/main/resources/com/microsoft/jenkins/containeragents/aci/AciContainerTemplate/config.jelly

```diff
@@ -49,6 +49,15 @@
                                     deleteCaption="${%Delete_Volume}"/>
     </f:entry>
 
+    <f:optionalBlock name="privateIpAddress" title="${%Use_Private_IP}" field="privateIpAddress" checked="${!empty instance.privateIpAddress}">
+            <f:entry title="Name of the VNET" >
+                <f:textbox field="vnet" value="${instance.privateIpAddress.vnet}"/>
```

Comment:
you should not need to set the value likely swapping to optional property would fix this